_posts/2018-01-02-twitter-cards-for-jekyll-with-jekyll-seo-tag.md

```diff
@@ -34,6 +34,11 @@ This will make GitHub pages know to use the plugin.
 
 To install locally run `gem install jekyll-seo-tag` at the command line.
 
+Finally, add the following tag right before `</head>` in your site's template(s):
+```
+{% seo %}
```

Comment:
This needs to be wrapped in `{% raw %}` or else it will Jekyll will actually think it should generate and output the `{% seo %}` tag here...

```
{% raw %}{% seo %}{% endraw %}
```